docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,179 @@
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use TileData to switch between data on charts"
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use for page-level navigation, as TileData is to be used in conjunction with a chart. Use [Tabs](/web/tabs), [SegmentedControl](/web/segmentedcontrol), [SideNav](/web/sidenavigation), or another local navigation instead."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.displayName}
+        description="Users should be able to navigate or activate a TagData using a keyboard or other input modalities. Be sure to include an `accessibilityLabel` for the screen reader within the child element."
+      />
+
+      <MainSection
+        name="Localization"
+        description="Make sure content that is placed in a data point is set-up to work in R-T-L languages. Be sure to localize text and `accessibilityLabel` in TileData and all subcomponents as well. When the text of the TileData reaches its Max width, either intentionally or through localization, will wrap as needed to display the full text. Keep this in mind when selecting wording for your TileData menu items. Note that localization can lengthen text by 20 to 30 percent. "
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`Use TileData's Data Visulization to display multiple colors`}
+          title="Colors"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={color} name="Colors Variant" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description="TileData can be used with a tooltip to display clarifying information"
+          title="Tooltip"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={tooltip} name="Tooltip variant" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description="Removes interactivity from the element"
+          title="Disabled"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={disabled} name="Disabled variant" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description={`Use checkboxes when enabling a multi-select experience. You can manage state by passing the \`selected\` prop`}
+          title="Group"
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={group} name="Checkbox Variant" />}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <QualityChecklist component={generatedDocGen?.displayName} />
+
+      <MainSection name="Related">
```

Comment:
What other similar components are there? Either visually similar or functionally similar?

Reply:
> 

Hey Ryan! we don't have visually similar components ATM in Gestalt. but functionality is similar to checkboxes/radios

Reply:
@rlingineni Can we add Checkbox and RadioGroup here?